ui/src/CurrentUser.ts

```diff
@@ -99,7 +99,9 @@ export class CurrentUser {
     };
 
     public tryAuthenticate = async (): Promise<AxiosResponse<IUser>> => {
```

Comment:
The plugin detail page threw an exception because the plugin data wasn't loaded there.